core/src/main/scala/cats/Applicative.scala

```diff
@@ -13,13 +13,21 @@ import simulacrum.typeclass
  * Must obey the laws defined in cats.laws.ApplicativeLaws.
  */
 @typeclass trait Applicative[F[_]] extends Apply[F] { self =>
+
   /**
-   * `pure` lifts any value into the Applicative Functor
+   * `pure` lifts any value into the Applicative Functor.
    *
    * Applicative[Option].pure(10) = Some(10)
    */
   def pure[A](x: A): F[A]
 
+  /**
+   * `pureEval` lifts any value into the Applicative Functor.
+   *
+   * This variant supports optional laziness.
+   */
+  def pureEval[A](x: Eval[A]): F[A] = pure(x.value)
```

Comment:
Making the default delegate to `pure` would negate any laziness properties of the `Eval` instance we pass in right? Oh - the intention would be to override as necessary, say when defining it for a `Task`-y thing?